schutzbot/deploy.sh

```diff
@@ -123,6 +123,12 @@ for SOURCE in $(sudo composer-cli sources list); do
     sudo composer-cli sources info "$SOURCE"
 done
 
+# Currently openstack/rhel-8.4-x86_64 cannot subcribe, subscription is disabled.
+# Can not pull from remote repo, use my own user/pass here for test.
+# Should not expose the account here, will replace them with test team account
+# Remove this after openstack/rhel-8.4-x86_64 can subscribe
+sudo podman login docker.io -u yihredhat -p p0o9i8u7Ginker1!
```

Comment:
Login/Password exposed in plain text. Change those immediately!

Reply:
Because subscription is disabled, when start Postgres container, it failed to connect to docker.io, that's why I asked is there a test account that I can use. If not, I need your help to set login/password as project env variables that I can use them.

Reply:
Fixed, thanks.